migrations/0115.sql

```diff
@@ -0,0 +1,89 @@
+ALTER TABLE stock_log
+    RENAME TO stock_log_old;
+
+CREATE TABLE stock_log(
```

Comment:
Was this only about adding the new column `user_id`? Then just `ALTER TABLE stock_log ADD user_id INTEGER NOT NULL` would be enough...